includes/class-sensei-lesson.php

```diff
@@ -5359,6 +5360,39 @@ public function has_sensei_blocks( $lesson = null ) {
 
 	}
 
+	/**
+	 * Register the meta fields for the lesson post type.
+	 *
+	 * @return void
+	 */
+	public function set_up_meta_fields() {
+		register_post_meta(
+			'lesson',
+			'_initial_content',
+			[
+				'show_in_rest'  => true,
+				'single'        => true,
+				'type'          => 'string',
+				'auth_callback' => [ $this, 'post_meta_auth_callback' ],
+			]
+		);
+	}
+
+	/**
+	 * Check if user can update lesson meta.
+	 *
+	 * @internal
+	 *
+	 * @param bool   $allowed  Whether the user can add the meta.
+	 * @param string $meta_key The meta key.
+	 * @param int    $post_id  The post ID where the meta key is being edited.
+	 *
+	 * @return boolean Whether the user can edit the meta.
+	 */
+	public function post_meta_auth_callback( $allowed, $meta_key, $post_id ) {
+		return current_user_can( 'edit_post', $post_id );
```

Comment:
Maybe this is slightly better?

```suggestion
		return current_user_can( 'edit_lessons', $post_id );
```

Reply:
Sounds good, I've updated it here https://github.com/Automattic/sensei/pull/7465/commits/661e3456c768e57c0e577d602b2ad1d0a740b22a , Thanks!

Also I've made an update in the Zoom/Meeting and Files to Download patterns here https://github.com/Automattic/sensei/pull/7465/commits/c83ae7eae59f966a36774fdd15c1e1dd682ff361